xarray/tests/test_backends.py

```diff
@@ -992,6 +1032,80 @@ def test_encoding_unlimited_dims(self):
             with pytest.warns(UserWarning):
                 ds.to_netcdf(tmp_file, engine='h5netcdf', unlimited_dims=['y'])
 
+# tests pending h5netcdf fix
+#class H5NetCDFDataTestAutocloseTrue(H5NetCDFDataTest):
+#    autoclose = True
+
+class OpenMFDatasetTest(TestCase):
+    autoclose = True
```

Comment:
I don't think you need this class variable, since this test class does not use inheritance.